### PR TITLE
fpg SOR integration

### PR DESCRIPTION
**Pull request by dev1-fund3**
TODO:
- [x] error cases for create_SOR_order
- [x] figure out best way to retrieve fpg api keys for public client (distribute them among clients in json file or retrieve from fpg creds from s3 (but that would require s3 creds we'd need to distribute), other ideas?) - distribute api keys to clients

### CODE REVIEW

**Comment by dev1-fund3** [comment]
@calvinleungyk what do you think?
what's best way to retrieve fpg api keys for public client:
a. distribute them among clients in json file or 
b. retrieve from fpg creds from s3 (but that would require s3 creds we'd need to distribute), 
c. other ideas?

File: omega_client/fpg/fpg_lib.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    logger.info('json to be sent to fpg', extra={'sor_body': sor_body})
+    r = requests.post(ORDERS_URL, json=sor_body, auth=auth)
+    status_code, resp = r.status_code, r.json()
+    logger.info('fpg response status code', extra={'status_code': status_code})
+    logger.info('fpg response body', extra={'response': resp})
+
+    orders = []
+    error_message = ''
+    if status_code == 200:  # successful response
+        fpg_order_list = resp.get('createOrderResponse', []).get(
```
Comment (calvinleungyk) [inline omega_client/fpg/fpg_lib.py]: Suggest to wrap `fpg_order_list` into a function named `def process_fpg_response(fpg_order_list)` and returns a `f3_orders_list`

File: omega_client/fpg/fpg_lib.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    :param auth: (FPGAuth) authentication used to sign request
+    :return: List[Order] list of child Orders to be placed (empty list [] if
+    status code is not 200), status_code (int), error_message (str)
+    """
+    base, quote = order.symbol.split('/')[0], order.symbol.split('/')[1]
+    algo = 'DMA'
+    sor_body = {
+        "base": base,
+        "quote": quote,
+        "size": float(order.quantity),
```
[the remark this answers is not in the document]
Comment (dev1-fund3) [reply omega_client/fpg/fpg_lib.py]: We can submit floats, but we get ints returned

File: omega_client/fpg/fpg_test.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import json, hmac, hashlib, time, requests
```
[the remark this answers is not in the document]
Comment (dev1-fund3) [reply omega_client/fpg/fpg_test.py]: I'll move it to examples

**Pull request merged**